.flake8

```diff
@@ -1,6 +0,0 @@
-[flake8]
-count = True
-statistics = True
-max-line-length = 88
```

Comment:
let's make sure max line length is still respected in the new ruff config

Reply:
Check the main comment (https://docs.astral.sh/ruff/configuration/#__tabbed_1_1): if not specified ruff uses 88